docs/dev/designs/RestFrontEnd.md

```diff
@@ -0,0 +1,136 @@
+# REST Front-end
```

Comment:
I do not suggest to use word front-end, since typically this refers to web app GUI. All server side stuff is always called back-end, however in our architecture we can say that this is "front" client-seeing process. In this case, I'd suggest to change word front-end just to "server".